web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -1232,17 +1232,15 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
       paddedHeight = this.computedAdjustedOskHeight(this.height);
     }
 
-    let b = this.layerGroup.element as HTMLElement;
     let gs = this.kbdDiv.style;
-    let bs = b.style;
```

Comment:
The choice of variable names here wasn't particularly helpful.

We want to leave the layer-group font-size alone here - that's where the touch-layout fontsize property is applied, and why it was being overwritten.